src/wp-includes/speculative-loading.php

```diff
@@ -0,0 +1,245 @@
+<?php
+/**
+ * Speculative loading functions.
+ *
+ * @package WordPress
+ * @subpackage Speculative Loading
+ * @since 6.8.0
+ */
+
+/**
+ * Returns the speculation rules configuration.
+ *
+ * @since 6.8.0
+ *
+ * @return array<string, string>|false Associative array with 'mode' and 'eagerness' keys, or false if speculative
+ *                                     loading is disabled.
```

Comment:
```suggestion
 * @return array<string, string>|null Associative array with 'mode' and 'eagerness' keys, or null if speculative
 *                                    loading is disabled.
```

Reply:
Updated in https://github.com/WordPress/wordpress-develop/pull/7860/commits/cc66024af13833c2a4ae1704e6de8a49ca766feb